include/libtorrent/announce_entry.hpp

```diff
@@ -57,7 +57,7 @@ TORRENT_VERSION_NAMESPACE_2
 	struct TORRENT_EXPORT announce_infohash
 	{
 		// internal
-		TORRENT_UNEXPORT announce_infohash();
+		announce_infohash();
```

Comment:
this is needed because below in `announce_endpoint`
```
aux::array<announce_infohash, num_protocols, protocol_version> info_hashes;
```

Reply:
would it work to, instead, add a constructor to `announce_endpoint`, and define it as `= defailt` in announce_endpoint.cpp?

Reply:
yes, it did work, changed and squashed